docs/section5/Rule5-49.md

```diff
@@ -0,0 +1,53 @@
+
+# Envelope - Rule 5-49  
+
+**Rule ID:** 5-49  
+**Rule Description:** The proposed air leakage rate of the building envelope (I75Pa) at a fixed building pressure differential of 0.3 in. of water shall be 0.6 cfm/ft2 for buildings providing verification in accordance with Section 5.9.1.2. The air leakage rate of the building envelope shall be converted to appropriate units for the simulation program using one of the methods in Section G3.1.1.4. Exceptions: When whole-building air leakage testing, in accordance with Section 5.4.3.1.1, is specified during design and completed after construction, the proposed design air leakage rate of the building envelope shall be as measured.  
+**Rule Assertion:** Sum of P-RMR zone.infiltration.air_leakage_rate = expected value  
+**Appendix G Section:** Section G3.1-5(b) Building Envelope Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** 
+
+  1. get_surface_conditioning_category()
+  2. get_zone_conditioning_category()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for P_RMR: `scc_dict_p = get_surface_conditioning_category(P_RMR)`
+
+- Get zone conditioning category dictionary for P_RMR: `zone_conditioning_category_dict_p = get_zone_conditioning_category(P_RMR)`
+
+- For each building segment in the Proposed model: `for building_segment_p in P_RMR.building.building_segments:`
+
+  - For each thermal block in building segment: `for thermal_block_p in building_segment_p.thermal_blocks:`
+
+    - For each zone in thermal block: `for zone_p in thermal_block_p.zones:`
+
```

Comment:
Please remove "thermal block" nomenclature, so that the RDS starts with looping through zones similar to 5-47

Reply:
Updated.